libioc/Provisioning/__init__.py

```diff
@@ -84,7 +84,8 @@ def __available_provisioning_modules(
 		self
 	) -> typing.Dict[str, Prototype]:
 		return dict(
-			ix=libioc.Provisioning.ix
+			ix=libioc.Provisioning.ix,
+			puppet=libioc.Provisioning.puppet
```

Comment:
we could split this to into `puppet-apply` and `puppet-server` (or `puppet-agent`?)